scripts/ensure-flow.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eu
+comm -13 -z \
+  <(git grep -z --name-only -e @flow -e @no-flow | sort -z) \
+  <(find src config scripts -name '*.js' -print0 | sort -z) \
```

Comment:
I note that there are a number of places where we implicitly depend on all "non-`node_modules`"/"relevant" code being in `{src,config,scripts}`. No change requested, but (a) moving forward, we might want to prefer explicitly excluding `node_modules`, because (b) if we change this invariant then we will have to remember to fixup a bunch of places in the code.

Reply:
A good point.